### DISCUSSION TITLE
[M-Z] (22 Gennaio 2019) Esercizio 1

### DISCUSSION THREAD

**Discussion by rimaout** [sapienzastudentsnetwork/metodi-matematici-per-l-informatica#432] [Esami & Esoneri M-Z]
## Traccia

<img width="589" alt="image" src="https://github.com/user-attachments/assets/3e6c7cba-7e53-466c-a1af-a18032d2785e">

>**Stesso esercizio:** #423 (15 Gennaio 2018 - Esercizio 1)

**Comment by rimaout** [comment]
<img width="500" alt="image" src="https://github.com/user-attachments/assets/f22524ce-da08-4db1-8878-5ec844eb507f">

**Comment by matypist** [comment]
Questo esercizio è un duplicato di [[M-Z] (15 Gennaio 2018) Esercizio 1 #423](https://github.com/sapienzastudentsnetwork/metodi-matematici-per-l-informatica/discussions/423) , per evitare una frammentazione delle proposte di soluzione potete quindi scrivere/leggere eventuali possibili soluzioni direttamente lì
